mlos_bench/mlos_bench/schedulers/parallel_scheduler.py

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""A simple single-threaded synchronous optimization loop implementation."""
```

Comment:
update docstring

Reply:
Also, please add some more complex docstrings that explain what this is module can help with and how it does that that include some executable usage details of the config insantiation at least.  There should be some other examples elsewhere in the source tree that you can crib off of or I can help cook up an example in abit.